### DISCUSSION TITLE
How to change from COLMAP coordinate system to Open3d Coordinate system

### DISCUSSION THREAD

**Discussion by Dharmendra04** [isl-org/Open3D#6649] [Q&A]
I have obtained a sparse point cloud from COLMAP and Used NKSR: https://github.com/nv-tlabs/NKSR to create a mesh reconstruction. I also obtained the surface normals for each mesh vertices using open3d: o3d_mesh.compute_vertex_normals() code.

I want to know whether the output normals will be in the open 3D world coordinate system or in the COLMAP coordinate system. If it is in the open3d world coordinate system(I guess so), how I can change the normals back to the COLMAP coordinate system?

**Comment by rwijmashell** [comment]
In my experience, the coordinate systems are the same.
